lambdas/functions/clusterize/clusterize.go

```diff
@@ -99,6 +91,25 @@ func doClusterize(p ClusterizationParams, funcDef functions_def.FunctionDef) (cl
 		return
 	}
 
+	// check if admin password value is set - if not, set it
+	if _, err = common.GetSecret(p.AdminPasswordId); err != nil && err.(awserr.Error).Code() == secretsmanager.ErrCodeResourceNotFoundException {
```

Comment:
in the new code, in what case it can already be set and we should use it and not re-create?
in case clusterization failed?

Reply:
but on upgrade we don't go through clusterize, it was already done